Doc/c-api/init.rst

```diff
@@ -1904,6 +1904,52 @@ Python-level trace functions in previous versions.
 
 .. versionadded:: 3.12
 
```

Comment:
I suggest to add a new section and just put the "versionadded:: 3.13" just after.

Example: "Reference Tracing".

Reply:
👍